source/_sass/infeedl.scss

```diff
@@ -63,9 +63,16 @@ h1, h2, h3, h4, h5, h6 {
   li {
     a {
       text-transform: none;
-      color: #fff;
+      color: #F5A623;
```

Comment:
Color literals like `#F5A623` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F5A623` should be written as `#f5a623`